ghost/core/test/unit/frontend/web/ghost-stats.test.js

```
@@ -0,0 +1,683 @@
+      
+      // Check that an XMLHttpRequest was made
+      const xhrInstance = env.lastXHR();
+      expect(xhrInstance).to.exist;
+      expect(xhrInstance.method).to.equal('POST');
+      
+      // Check that the request data contains page hit data
+      const requestData = JSON.parse(xhrInstance._data);
+      expect(requestData.action).to.equal('page_hit');
+    });
+
+    it('should track page hits on history pushState', function () {
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)